### PR TITLE
Environment Variables documentation update

### PR DESCRIPTION
**Pull request by isaac73**
# Type of change? 

- [ ] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 

This PR adds documentation for dependent environment variables

 [Equivalent DotNet PR]( https://github.com/Azure/pcs-config-dotnet/pull/77)

**Checklist:**

- [x] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [x] I have updated the documentation accordingly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,12 +19,23 @@ How to use the microservice
...
+Run `scripts\env-vars-setup.cmd` on Windows or `source scripts/env-vars-setup`
 on Mac/Linux to set up the environment variables needed to run the service locally.
-If using environment variables, this service requires the following environment
+In Windows you can also set these [in your system][windows-envvars-howto-url].
+
+If using envornemnt variables, this service requires the following environment
 variables to be set:
 - `PCS_STORAGEADAPTER_WEBSERVICE_URL` - the url for
-  the [Storage Adapter Webservice](https://github.com/Azure/pcs-storage-adapter-java)
+  the [Storage Adapter Webservice](https://github.com/Azure/pcs-storage-adapter-dotnet)
```
Comment (jillcary) [inline README.md]: Replace with the Java version `https://github.com/Azure/pcs-storage-adapter-java`

File: README.md [hunk 2]
```diff
@@ -19,12 +19,23 @@ How to use the microservice
...
+  used for key value storage
+- `PCS_TELEMETRY_WEBSERVICE_URL` - the url for
+  the [Telemetry Webservice](https://github.com/Azure/device-telemetry-dotnet.git)
```
Comment (jillcary) [inline README.md]: Same comment, refer to the java version of the telemetry repo

Comment (jillcary) [reply README.md]: (also note the simulation service only has a dotnet version, so that one is fine)

File: README.md [hunk 3]
```diff
@@ -19,12 +19,23 @@ How to use the microservice
...
+  used for key value storage
+- `PCS_DEVICESIMULATION_WEBSERVICE_URL` - the url for
+  the [Device Simulation Webservice](https://github.com/Azure/device-simulation-dotnet.git)
+  used for key value storage
+- `PCS_IOTHUBMANAGER_WEBSERVICE_URL` - the url for
+  the [IOT Hub Manager Webservice](https://github.com/Azure/iothub-manager-dotnet.git)
```
Comment (jillcary) [inline README.md]: Java version here as well

**Comment by jillcary** [comment]
Please add a PR description above and fill out the template with the type of change. It may also be helpful to refer to the equivalent PR in the dotnet repo: https://github.com/Azure/pcs-config-dotnet/pull/77/files

**Comment by isaac73** [comment]
This PR adds documentation for dependent environment variables

Equivalent DotNet PR - [https://github.com/Azure/pcs-config-dotnet/pull/77](url)

**Pull request merged**